### ISSUE TITLE
SIA-R10 and autocomplete="one-time-code"

### ISSUE THREAD

**Issue by dan-tripp-siteimprove** [Siteimprove/alfa#1515]
This code fails SIA-R10: 
`<input  type="text"  autocomplete="one-time-code" >`
I think that it should pass, because "one-time-code" is a valid value according to https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#autofill-detail-tokens

**Comment by Jym77** [comment]
Good catch. Looks like this has been added since the rule was written 🤔
